hugegraph-postgresql/src/main/java/com/baidu/hugegraph/backend/store/postgresql/PostgresqlSessions.java

```diff
@@ -45,6 +46,20 @@ public PostgresqlSessions(HugeConfig config, String database, String store) {
         super(config, database, store);
     }
 
+    @Override
+    public boolean existsDatabase() {
+        String statement = String.format(
+                           "SELECT datname FROM pg_catalog.pg_database " +
+                           "WHERE datname = '%s';", this.database());
```

Comment:
Please escape the quotes that may be contained in the database name, prepare for future support for dynamic creation of graph.
https://blog.csdn.net/github_36032947/article/details/78442189

To prevent this problem fundamentally, we can let this.database() method return the name that has been escaped(or add a databaseEscaped() method).